### DISCUSSION TITLE
systemd worker not activating

### DISCUSSION THREAD

**Discussion by I-Sarma-I** [pocketbase/pocketbase#5478] [Q&A] [answered]
`im trying to setup starting systemd worker but i keep getting the same erorr`

**[Unit]
Description = pocketbase
[Service]
Type           = simple
User           = root
Group          = root
LimitNOFILE    = 4096
Restart        = always
RestartSec     = 5s
StandardOutput = append:/root/pb/errors.log
StandardError  = append:/root/pb/errors.log
ExecStart      = /home/pocketbase/pocketbase serve --http <ip>
[Install]
WantedBy = multi-user.target**

`Here is my systemd setup and my error`

**pocketbase.service - pocketbase
     Loaded: loaded (/lib/systemd/system/pocketbase.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Tue 2024-09-03 18:40:02 UTC; 2s ago
    Process: 63010 ExecStart=/home/pocketbase/pocketbase serve --http <ip> (code=exited, status=>
   Main PID: 63010 (code=exited, status=209/STDOUT)
        CPU: 918us**

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that the `/root/pb/` directory exists (or move the logs in your `/home/pocketbase` dir).

**Reply by I-Sarma-I** [reply]
moving the logs to the `home/pocketbsae` directory fixed the problem. tnx 🔥
